### PR TITLE
Core/minor updates tests

### PR DESCRIPTION
**Pull request by philbucher**
Just that

see how the tests can be skipped now :)

### CODE REVIEW

File: kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp [hunk 1]
```diff
@@ -99,17 +105,15 @@ namespace Kratos
         */
         KRATOS_TEST_CASE_IN_SUITE(TestIntegrationValuesExtrapolationToNodesProcessTriangle, KratosCoreFastSuite)
         {
+            KRATOS_SKIP_TEST_IF_NOT(KratosComponents<Element>::Has("UpdatedLagrangianElement2D3N")) << "This test needs the StructuralMechanicsApplication" << std::endl;
```
Comment (loumalouomega) [inline kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:108]: Since when do we have this?

**Review by loumalouomega — APPROVED** [review]

File: kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp (hunk 1 above)
Comment (philbucher) [reply kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:108]: #5929

Comment (loumalouomega) [reply kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:108]: imagen

Comment (philbucher) [reply kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:108]: now we only need a proper test in Cpp if an app is imported

Comment (loumalouomega) [reply kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:108]: I will update the Meshing with the new mechanism

Comment (loumalouomega) [reply kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp:108]: See #6681

**Pull request merged**